x/pki/tests/test-design.md

```diff
@@ -0,0 +1,276 @@
+## [Add DA Root](./handler_add_paa_cert_test.go)
+
+### Propose adding of DA root certificate
+
+Indexes to check:
+
+* Present:
+    * `ProposedCertificate`
+    * `UniqueCertificate`
+* Missing:
+    * `RejectedCertificate`
```

Comment:
What about Revoked?

Reply:
Is it possible to restore revoked DA root certificate?
Right now I see that none of `ProposeAddX509RootCert` / `ApproveAddX509RootCert` message handlers interacts with revoked certificate indexes.

Reply:
Following this logic we also should check all noc indexes and so on.